src/main/java/knightminer/simplytea/core/Registration.java

```diff
@@ -3,24 +3,12 @@
 import knightminer.simplytea.SimplyTea;
 import knightminer.simplytea.block.TeaSaplingBlock;
 import knightminer.simplytea.block.TeaTrunkBlock;
-import knightminer.simplytea.item.CocoaItem;
-import knightminer.simplytea.item.HotTeapotItem;
-import knightminer.simplytea.item.TeaCupItem;
-import knightminer.simplytea.item.TeaStickItem;
-import knightminer.simplytea.item.TeapotItem;
-import knightminer.simplytea.item.TooltipItem;
-import knightminer.simplytea.item.WoodBlockItem;
+import knightminer.simplytea.item.*;
```

Comment:
Please don't use wildcard imports, they make the code a lot harder to manually trace on GitHub